var/spack/repos/builtin/packages/r/package.py

```diff
@@ -82,6 +82,8 @@ class R(Package):
     depends_on('pcre')
     depends_on('jdk')
 
+    patch('zlib.patch', when='@:3.3.1', level=0)
```

Comment:
*bzzt...*  😝

```
[hartzelg@spack-0 spack-r-fix]$ /rss/linuxbrew/bin/ack 'patch.*level' var/spack/repos/builtin/packages/
var/spack/repos/builtin/packages/fftw/package.py
42:    patch('pfft-3.3.5.patch', when="@3.3.5+pfft_patches", level=0)
43:    patch('pfft-3.3.4.patch', when="@3.3.4+pfft_patches", level=0)

var/spack/repos/builtin/packages/bowtie2/package.py
35:    patch('bowtie2-2.5.patch', when='@2.2.5', level=0)

var/spack/repos/builtin/packages/mpibash/package.py
36:    patch('mpibash-4.3.patch', level=1, when='@4.3')

var/spack/repos/builtin/packages/nwchem/package.py
73:            patch(url, when=condition, level=0, md5=md5)

var/spack/repos/builtin/packages/r/package.py
85:    patch('zlib.patch', when='@:3.3.1', level=0)

var/spack/repos/builtin/packages/go/package.py
70:    patch('misc-cgo-testcshared.patch', level=0, when='@1.6.4:1.7.4')

var/spack/repos/builtin/packages/fastqc/package.py
41:    patch('fastqc.patch', level=0)
```

(that's how I figured out the option).

As a similarly-detail-oriented person, I'll go with the majority.  Sit tight.